xpdacq/yamlclass.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+#
+# xpdacq            by Billinge Group
+#                   Simon J. L. Billinge sb2896@columbia.edu
+#                   (c) 2016 trustees of Columbia University in the City
+#                   of
+#                        New York.
+#                   All rights reserved
+#
+# File coded by:    Timothy Liu
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+
+import os
+import yaml
+
+class YamlClass:
```

Comment:
Sorry, never mind this comment I forgot a bunch of things about how python objects work.